feture-1/one-more-file.txt

```diff
@@ -0,0 +1,4 @@
+This file was also edit in a feature-1 branch
```

Comment:
It's really good that you have decided to add this line of code

Reply:
Thanx for this coment